src/main/java/com/mastery/java/task/dto/validator/IsGender.java

```diff
@@ -0,0 +1,18 @@
+package com.mastery.java.task.dto.validator;
+
+import javax.validation.Constraint;
+import javax.validation.Payload;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+
+@Constraint(validatedBy = IsGenderValidator.class)
+@Target( { ElementType.METHOD, ElementType.FIELD })
+@Retention(RetentionPolicy.RUNTIME)
+public @interface IsGender {
```

Comment:
strange interface name

Reply:
this is a custom annotation for gender validation
should it look different? please tell me if this is the case

Reply:
OK let's discuss this internally